ddtrace/internal/core/crashtracking.py

```diff
@@ -57,6 +59,9 @@ def start() -> bool:
     for key, value in crashtracker_config.tags.items():
         add_tag(key, value)
 
+    if profiling_config.enabled:
+        add_tag("profiler_config", config_str(profiling_config))
```

Comment:
can profiling be enabled/disabled at runtime via remote config?

Reply:
I am not aware of such functionality. @sanchda do we have this?

Reply:
not a problem, was mostly trying to explore if we would end up in a state where we don't set this config but profiling is enabled.

as long as it is startup only decision, then I think we are good here?